plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/CubridSQLFormatter.java

```diff
@@ -0,0 +1,374 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.cubrid;
+
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.Set;
+import java.util.StringTokenizer;
+
+import org.jkiss.code.NotNull;
+
+public class CubridSQLFormatter

```

Comment:
So, all this ample and complex class is just for View DDL formatting?

Reply:
Yes, it is.

Reply:
Mmm, now, we already have `SQLFormatUtils.formatSQL(getDataSource(), definition));` (like in MySQL bundle)
Did you try it? Can you use it instead of this class? 
This class is a whole of hacks, non-private variables, hardcoded strings, etc. We can't accept it in this form without severe reasons.

Reply:
Thank you for your suggestion, I will try this